thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -130,7 +139,7 @@ export default Ember.Component.extend({
 
     toggleHidden() {
       const { isHidden } = this.getProperties('isHidden');
-      this.setProperties({ isHidden: !isHidden });
+      this.setProperties({ isHiddenUser: !isHidden });
```

Comment:
you can also do `this.toggleProperty('isHidden')`

Reply:
the one-way interaction between isHidden and isHiddenUser is important here